pkg/kubelet/pleg/generic.go

```diff
@@ -264,6 +264,19 @@ func (g *GenericPLEG) relist() {
 		}
 		// Update the internal storage and send out the events.
 		g.podRecords.update(pid)
+
+		// Create containerExitCode map for lookup
+		containerExitCode := make(map[string]int)
```

Comment:
should this be lazily populated on demand? Is it correct understanding that this is done here to reuse in multiple events, but it will not be used if those events are all starts of containers?

Reply:
you are right. Will change it to lazily populated only it occurs containerDied event for the first time.